src_py/__init__.py

```diff
@@ -77,13 +77,12 @@ def warn(self):
 
 # we need to import like this, each at a time. the cleanest way to import
 # our modules is with the import command (not the __import__ function)
-# isort: skip_file
```

Comment:
why is this removed?

Reply:
I am not sure. Maybe someone changed that line in the two days before I made more changes and I overrode it.
I will add it back later.